devfilev2.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2022 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+schemaVersion: 2.1.0
+metadata:
+  name: che-docs
+attributes:
+  controller.devfile.io/storage-type: ephemeral
```

Comment:
The ephemeral option is a very dangerous one for contributors. You loose all your contributions if for any reason you cannot commit and push before the workspace stop. Are you sure we want this? I find the `generateName` option, creating a new workspace when you don't restart an existing workspace less dangerous.